### PR TITLE
Start tx validator v2, and fix bug in zkSync validations

### PR DESCRIPTION
**Pull request by mds1**
Creates a new validator that will be updated as necessary going forward. Also fixes a bug in the zkSync validator.

cc @owocki

### CODE REVIEW

File: app/economy/tx.py [hunk 1]
```diff
@@ -294,19 +482,26 @@ def get_token_originators(to_address, token, from_address='', return_what='trans
     endpoint = 'token-transfers' if token != '0x0' else 'ether-transfers'
     url = f'https://api.aleth.io/v1/{endpoint}?filter[to]=' + address + '&filter[token]=' + token + '&page%5Blimit%5D=100'
     if token == '0x0':
+        print('qqqq')
         url = f'https://api.aleth.io/v1/{endpoint}?filter[account]=' + address + '&page%5Blimit%5D=100'
     if from_address:
+        print('wwww')
```
Comment (thelostone-mc) [inline app/economy/tx.py:488]: ^ do we wanna remove all the print stmt :P

Comment (octavioamu) [inline app/economy/tx.py:488]: do we need the prints?

**Review by octavioamu — CHANGES_REQUESTED** [review]

File: app/economy/tx.py (hunk 1 above)
Comment (mds1) [reply app/economy/tx.py]: Oops, yea those were leftover from my expert debugging techniques. Removed in f61690e24012169ba8d24a85fb99dc21edf26e24

Comment (mds1) [reply app/economy/tx.py]: Oops, yea those were leftover from my expert debugging techniques. Removed in f61690e24012169ba8d24a85fb99dc21edf26e24

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**